pkg/reconciler/managed/reconciler.go

```diff
@@ -72,6 +72,8 @@ const (
 	reasonDeleted event.Reason = "DeletedExternalResource"
 	reasonCreated event.Reason = "CreatedExternalResource"
 	reasonUpdated event.Reason = "UpdatedExternalResource"
+
+	reasonNeedReconcile event.Reason = "NeedReconcileManagedResource"
```

Comment:
I'm not sure about this event name - technically we're already reconciling the managed resource. 🤔 What about something like `DriftDetected` or `UpdateNeeded`?

Reply:
Another idea might be to include this with the `UpdatedExternalResource` event we already send.